src/components/RoomView/RoomView.tsx

```diff
@@ -193,10 +191,13 @@ const RoomView = (props: ClassName) => {
       {/* is there isnt at least one person to each role, setup board should be disabled / not visible */}
       {/* is host AND there is at least one person on each team */}
       {isHost && (
+          <SetupGame />
+      )}
+      {/* {isHost && (
```

Comment:
Why not delete this?

Reply:
was for testing and forgot to delete! thx for catching